data/quarg missions.txt

```diff
@@ -72,6 +73,7 @@ mission "First Contact: Kuwaru Efreti"
 	to offer
 		has "First Contact: Quarg: offered"
 	on offer
+		log "Made contact with a faction of Korath called the Efreti, which means Friends. They don't seem to be hostile, and have the protection of the Quarg."
```

Comment:
Mm, yeah, I'm not a fan of "which means friends", as I instinctively try to parse that as a common noun, not a proper noun. My view is that since we're translating a word that's capitalised, it should be fine to write it as capitalised. After all, we call the Wanderers the Wanderers, not the wanderers, right?